CHANGELOG.asciidoc

```diff
@@ -17,6 +17,7 @@ https://github.com/elastic/beats/compare/v6.0.0-alpha2...master[Check the HEAD d
 - Rename `kubernetes` processor to `add_kubernetes_metadata`. {pull}4473[4473]
 - Rename `*.full.yml` config files to `*.reference.yml`. {pull}4563[4563]
 - The `scripts/import_dashboards` is removed from packages. Use the `setup` command instead. {pull}4586[4586]
+- Change format of the saved kibana dashboards to have a single JSON file for each dashboard {pull}4413[4413]
```

Comment:
It's by intention, as I forgot to update the changelog in the previous PR (#4413)